<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-TLB2RI3" ignored="0" name="testcases" tests="3" failures="1" timestamp="12 Dec 2020 08:40:57 GMT" time="0.072" errors="0">
  <testcase name="test01" time="0.024" classname="RetryFailedTestCase.FailedTestCases"/>
  <testcase name="test02" time="0.008" classname="RetryFailedTestCase.FailedTestCases"/>
  <testcase name="test03" time="0.0" classname="RetryFailedTestCase.FailedTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryFailedTestCase.FailedTestCases.test03(FailedTestCases.java:19)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test03 -->
</testsuite> <!-- testcases -->
